lnst/Devices/Device.py

```diff
@@ -339,7 +341,7 @@ def restore_original_data(self):
         try:
             self.restore_coalescing()
         except DeviceError:
-            pass
+            logging.warn(DeviceError)
```

Comment:
you used the class `DeviceError` here instead of the instance of the exception which you would do with:

```
except DeviceError as e:
    logging.warn(e)
```

it's also possible to use `logging.error` (https://docs.python.org/3/library/logging.html?highlight=logging%20exception#logging.exception) and that will automatically log the exception here... but in the level ERROR not WARN

Reply:
Good catch, will take a look and adjust.

Reply:
Done.